inventory/linux.cf

```diff
@@ -48,12 +61,19 @@ bundle common inventory_linux
   classes:
 
     any::
+      "have_proc_device_tree" expression => fileexists($(inventory_control.proc_device_tree));
```

Comment:
this seems a duplication of the same line in any.cf: https://github.com/cfengine/masterfiles/pull/2438/files#diff-e517c74d74557aab06f37a22ffb428372e0172c21f4293c38ac79ecafbfde986R980, maybe move this to controls/def.cf?